opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-converter.ts

```diff
@@ -36,3 +36,21 @@ export const convertIngestionTaxiiToStix = (instance: StoreEntityIngestionRss):
     }
   };
 };
+
+export const convertIngestionCsvToStix = (instance: StoreEntityIngestionCsv): StixIngestionCsv => {
+  const stixObject = buildStixObject(instance);
+  return {
+    ...stixObject,
+    name: instance.name,
+    description: instance.description,
+    uri: instance.uri,
+    csvMapper_id: instance.csvMapper_id,
```

Comment:
by convention, I think we use snake case in stix objects, so `csv_mapper_id` maybe?

Reply:
Indeed, I think you are right. Gonna change that